apps/users/forms/profile.py

```diff
@@ -134,40 +141,74 @@ def __init__(self, *args, **kwargs):
         self.helper.add_input(Submit("save_password", "Submit"))
 
 
-class PreferencesUpdateForm(forms.ModelForm):
-    class Meta:
-        model = Profile
-        fields = ["subscribed_newsletter", "subscribed_gmm_invite", "newsletter_recipient"]
+class PreferencesUpdateForm(forms.Form):
+    @staticmethod
+    def newsletter_field_name(newsletter):
+        return f"newsletter-{newsletter.slug}"
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, user: User, *args, **kwargs):
+        super().__init__(*args, initial={}, **kwargs)
+        self.user = user
         self.helper = FormHelper(self)
+        newsletters = Newsletter.objects.all()
```

Comment:
It is possible to create unpublished newsletter no? Might be good to filter for that here, so that we can create new ones, and keep them hidden and/or a test newsletter, without having it show up in the users profile.